src/test/java/net/sf/jabref/logic/cleanup/DatesCleanupTest.java

```diff
@@ -0,0 +1,205 @@
+package net.sf.jabref.logic.cleanup;
+
+import org.junit.Assert;
+import net.sf.jabref.model.entry.BibtexEntry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DatesCleanupTest {
+    private BibtexEntry entry;
```

Comment:
Instead of testing DatesCleanup, it would be better to test the formatter with simple input/output string pairs.